server/api/swagger/swagger.yaml

```diff
@@ -5749,17 +5771,18 @@ definitions:
         type: number
       increment_size:
         type: number
+      type:
+        type: string
   BrokerPairCreate:
     type: object
     required:
       - symbol
-      - buy_price
-      - sell_price
       - paused
       - user_id
       - min_size
       - max_size
       - increment_size
+      - type
```

Comment:
@fetok12 we should still add the fields we expect (swagger will validate their data type to avoid issues) but they are not required obviously.